wirelength_analyzer/wa.py

```diff
@@ -355,7 +354,7 @@ def add_all_nets_to_graph(self):
                 # Omit source (BELPins) that don't have any fanout
                 if len(branch.branches) == 0:
                     continue
-                if sl[branch.routeSegment.belPin.bel] in ('BUFCE'):
+                if sl[branch.routeSegment.belPin.bel] in ('BUFCE', 'BUFG_GT', 'BUFG_GT_SYNC'):
```

Comment:
Should this be moved into device data too?

Reply:
yes I think that would make sense, I'll change it

Reply:
https://github.com/Xilinx/fpga24_routing_contest/pull/40/commits/e7ac25f588ea425c97920bfa9c2d358f6aa1f5f3